src/workloads/query/OneMDocCollection_ObjectId.yml

```diff
@@ -0,0 +1,119 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This workload tests performance of IDHACK on queries with OID _id.
+
+KeyWords:
+- IDHACK
+- OID
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 2000
+
+GlobalDefaults:
+  Database: &Database test
+  Collection: &Collection Collection0
+  MaxPhases: &MaxPhases 5
+
+ActorTemplates:
+- TemplateName: IDHACK
+  Config:
+    Name: {^Parameter: {Name: "Name", Default: "IDHackQuery"}}
+    Type: CrudActor
+    Threads: {^Parameter: {Name: "Threads", Default: 1}}
+    Phases:
+      OnlyActiveInPhases:
+        Active: [{^Parameter: {Name: "OnlyActiveInPhase", Default: {unused: "please specify in which phases this actor should be active."}}}]
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          Repeat: 1000
+          Database: *Database
+          Collection: *Collection
+          Operations:
+          - OperationName: find
+            OperationCommand:
+              Filter: {_id: {^ObjectId: {^FormatString: {"format": "%06d%s", "withArgs": [{^RandomInt: {min: 0, max: 999999}}, "bbbbbbbbbbbbbbbbbb"]}}}}
```

Comment:
It seems like this is pretty legit, but could you run this locally with query stats enabled and check to see this query shapes average number of returned documents? Is it 1 - they all find something? Or is it lower?

Reply:
Both the max and min are 1! So looks like they all find something

Reply:
Awesome! Thanks for checking and a cool use of our new feature :D